Source/Resources/MongoDB/Resource.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Dolittle.Runtime.Lifecycle;
+using Dolittle.Runtime.ResourceTypes.Configuration;
+using MongoDB.Driver;
+
+namespace Dolittle.Runtime.Resources.MongoDB
+{
+    /// <summary>
+    /// Represents an implementation of <see cref="IResource"/>.
+    /// </summary>
+    [SingletonPerTenant]
+    public class Resource : IResource
+    {
+        readonly IConfigurationFor<Configuration> _configuration;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Resource"/> class. 
+        /// </summary>
+        /// <param name="configuration">The <see cref="IConfigurationFor{T}"/> <see cref="Configuration"/>.</param>
+        public Resource(IConfigurationFor<Configuration> configuration)
```

Comment:
So we don't need to access the resource type configuration in a special way?

Reply:
I was also thinking that, but I think that is handled bæin the background when getting the IConfogurationFor

Reply:
Yep - this is the same way we've been doing it for the EventStoreConfiguration :)